.gitignore

```diff
@@ -15,7 +15,6 @@ awsconfig
 .yarnrc
 .yarn/
 vendor/
-.eslintcache
```

Comment:
Question: should we keep the eslint cache in the version control? Feels like it should be excluded, right?

Reply:
this is already in gitignore so no need for it to be there twice. I've added it in my other pr because I missed the other one. 😅